automation_tools/repository.py

```diff
@@ -175,6 +174,12 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
         if sat_version == '6.3' and puppet4 == 'yes':
             repos.append('rhel-{0}-server-satellite-{1}-puppet4-rpms')
 
+    if sat_version in ['6.4', '6.5', '6.6']:
```

Comment:
we can remove this if condition as maintain is required for all sat versions.

Reply:
Thanks will do.